PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -1342,26 +1342,25 @@ bool FileManager::loadFileData(Document doc, const TCHAR * filename, char* data,
 	unsigned __int64 fileSize =_ftelli64(fp);
 	rewind(fp);
 	// size/6 is the normal room Scintilla keeps for editing, but here we limit it to 1MiB when loading (maybe we want to load big files without editing them too much)
-	unsigned __int64 bufferSizeRequested = fileSize + min(1<<20,fileSize/6);
-
+	unsigned __int64 bufferSizeRequested = fileSize + min(1<<20, fileSize/6);
 	
 	NppParameters& nppParam = NppParameters::getInstance();
 	NativeLangSpeaker* pNativeSpeaker = nppParam.getNativeLangSpeaker();
 
-	// As a 32bit application, we cannot allocate 2 buffer of more than INT_MAX size (it takes the whole address space).
-	// As a 64bit binary, we have more address for the allocation. However loading a 2GB file takes from 3 minutes to 7 minutes, which makes Notepad++ unusable:
-	// https://github.com/notepad-plus-plus/notepad-plus-plus/pull/11044
-	// So here we fix the size limit to 2GB
 	if (bufferSizeRequested > INT_MAX)

```

Comment:
Should this be changed as well?

Reply:
@mere-human 
It should be kept for 32bits binary.